example/WORKSPACE

```diff
@@ -1,62 +1,26 @@
-### RULES_PYTHON ###
-
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-RULES_PYTHON_NAME = "rules_python"
-
-RULES_PYTHON_TAG = "0.4.0"
-
-RULES_PYTHON_SHA = "954aa89b491be4a083304a2cb838019c8b8c3720a7abb9c4cb81ac7a24230cea"
-
-RULES_PYTHON_REPO = "bazelbuild"
-
-RULES_PYTHON_ARCHIVE = "tar.gz"
-
-RULES_PYTHON_URL = "https://github.com/{repo}/{name}/releases/download/{tag}/{name}-{tag}.{archive}".format(
-    name = RULES_PYTHON_NAME,
-    archive = RULES_PYTHON_ARCHIVE,
-    repo = RULES_PYTHON_REPO,
-    tag = RULES_PYTHON_TAG,
-)
-
-# use http_archive rule to load rules_python repo
+### RULES_PYTHON ###
```

Comment:
issue (non-blocking): this PR does 3 things at the same time - reformatting, upgrading rules_python, and using `local_repository` by default

suggestion: Move the formatting change and rules_python upgrade into separate follow-on PRs. It will be easier to review the changes separately, and the git history will be easier to understand. Additionally it will be easier to find a bug if `git bisect` is run.

Reply:
I thought each task only involves small change, that's why I put them in one PR with separated commits (commit message shows the detail). 
Thanks for the suggestion. I think it would definitely benefit the review and git bisect. I will do that in future PRs.